frontend/src/pages/TermPlanner/TermBox/TermBox.jsx

```diff
@@ -83,9 +75,11 @@ const TermBox = ({
               ),
             )}
             {provided.placeholder}
-            <S.UOCBadgeWrapper>
+            {console.log(theme)}
+            <S.UOCBadgeWrapper> 
               <Badge
-                style={{ backgroundColor: "#9254de" }}
+                className="UOCBadge"
+                style={{ backgroundColor: theme === "light" ? "#9254de" : "#51258f", boxShadow: "none" }} 
```

Comment:
Dont need to use the `theme` redux state. Instead, you can use the `useTheme` hook from styled components to get the `theme` object and set the `backgroundColor` :)